Generators/CombineGenerator.cs

```diff
@@ -35,6 +32,7 @@ private MemberDeclarationSyntax Combine(int count)
         {
             return MethodDeclaration(PredefinedType(Token(VoidKeyword)), "Combine")
                 .WithModifiers(TokenList(Token(TriviaList(Trivia(Documentation)), PublicKeyword, TriviaList())))
+                //.WithAttributeLists(SingletonList(AttributeList(SingletonSeparatedList(Attribute(IdentifierName("System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverage"))))))
```

Comment:
Did you mean to leave this?

Reply:
Not really, I had attempted to modify this code to use put the attribute on top of the method but with how the XML comments are attached to the public key word as trivia makes it hard for me to try and determine how to do that properly.